src/Summoner/Tree.hs

```diff
@@ -19,3 +22,28 @@ traverseTree (Dir name children) = do
     createDirectoryIfMissing False name
     withCurrentDirectory name $ for_ children traverseTree
 traverseTree (File name content) = writeFile name content
+
+-- | Pretty shows the directory tree content.
+showTree :: TreeFs -> Text
+showTree = unlines . showOne "  " "" ""
```

Comment:
Nice implementation! Probably we can do better in some future (to make output look like output of `tree` utilitly) but it requires more thinking.. And it's not important now.